bigtop-packages/src/common/oozie/patch5-HADOOP-11356.diff

```diff
@@ -0,0 +1,54 @@
+diff --git a/core/src/main/java/org/apache/oozie/action/hadoop/JavaActionExecutor.java b/core/src/main/java/org/apache/oozie/action/hadoop/JavaActionExecutor.java
+index 31d4817cb..c2233d1d8 100644
+--- a/core/src/main/java/org/apache/oozie/action/hadoop/JavaActionExecutor.java
++++ b/core/src/main/java/org/apache/oozie/action/hadoop/JavaActionExecutor.java
+@@ -45,13 +45,13 @@ import org.apache.hadoop.filecache.DistributedCache;
+ import org.apache.hadoop.fs.FileStatus;
+ import org.apache.hadoop.fs.FileSystem;
+ import org.apache.hadoop.fs.Path;
+-import org.apache.hadoop.fs.permission.AccessControlException;
```

Comment:
This was fixed by [OOZIE-2733](https://issues.apache.org/jira/browse/OOZIE-2733). Upgrading Oozie should be the right fix?